use optimization in D14020675

jspark1105 opened the pull request:
Summary: As title

Differential Revision: D14020769

protonu commented on caffe2/quantization/server/conv_dnnlowp_op.cc:
```diff
@@ -1223,7 +1223,9 @@ void ConvDNNLowPOp<T, ReluFused>::ConvNHWCCore_(
             in_qparams_[INPUT].zero_point,
             filter_zero_points_.data(),
-            row_offsets_.data() + tid * row_offset_size_per_thread,
+            filter_zero_points_[0]
```
if this is for symmetric quantization, is there any other way than to check the numeric value of the data?

jspark1105 replied:
Sorry can you elaborate more? What do you mean by numeric value of the data?

protonu replied:
I understood this as checking to see if the zero_point is 0 or not? If that is correct, I was wondering if there should be no storage at all. It's just nit, not a big issue.

jspark1105 replied:
You mean if zero_point is 0 then filter_zero_points_ should be empty? It's hard to do that in the current design because I believe we're also using the size of filter_zero_points_ to tell if we're using group-wise quantization.

jspark1105 committed:
use optimization in D14020675 (#16945)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16945

As title

Reviewed By: jianyuh

Differential Revision: D14020769

fbshipit-source-id: 7dedbf9ee23a6cafed450e76e9ccfa33ac766bee
